client.go

```diff
@@ -48,54 +47,54 @@ var (
 )
 
 const (
-	// DefaultAddress is the default IPV4 address that is used for a connection
-	DefaultAddress = "127.0.0.1"
-	// DefaultPort is the default port used for a connection
-	DefaultPort     = 6640
-	UNIX            = "unix"
-	TCP             = "tcp"
-	SSL             = "ssl"
-	SKIP_TLS_VERIFY = true
+	DefaultTCPAddress  = "127.0.0.1:6640"
+	DefaultUnixAddress = "/var/run/openvswitch/ovnnb_db.sock"
 )
 
-func ConnectDial(protocol string, target string) (net.Conn, error) {
-	if protocol == SSL {
-		cert, err := tls.LoadX509KeyPair(os.Getenv("CLIENT_CERT_CA_CERT"),
-			os.Getenv("CLIENT_PRIVKEY"))
+// Connect to ovn, using endpoint in format ovsdb Connection Methods
+// If address is empty, use default address for specified protocol
+func Connect(endpoints string, tlsConfig *tls.Config) (*OvsdbClient, error) {
+	var c net.Conn
+	var err error
+
+	for _, endpoint := range strings.Split(endpoints, ",") {
+		u, err := url.Parse(endpoint)
```

Comment:
Thanks for the update. However, OVSDB connect method format is not URL format. I.e. it is something like tcp:10.0.0.1:6641 instead of tcp://10.0.0.1:6641

Reply:
i handle this, check https://play.golang.org/p/7gdThaOe7-E

Reply:
OK, so it handles both. Thanks for explain. I was mislead by the test cases where "tcp://..." is used always. Please update the test cases to use "tcp:..." to avoid confusion, and please don't forget to update example/play_with_ovs.go.